src/Exceptionless.Core/Repositories/Configuration/ExceptionlessElasticConfiguration.cs

```diff
@@ -60,7 +60,9 @@ public sealed class ExceptionlessElasticConfiguration : ElasticConfiguration, IS
             ConfigureSettings(settings);
             foreach (var index in Indexes)
                 index.ConfigureSettings(settings);
-
+            if (!string.IsNullOrEmpty(Options?.UserName)) {
```

Comment:
We should change this to as we shouldn't need to null check it and we should verify both are set.

```cs

if (!String.IsNullOrEmpty(_options.UserName) && !String.IsNullOrEmpty(_options.Password))
    settings.BasicAuthentication(_options.UserName, _options.Password);
```

Reply:
I think what you mean is this or not ？
```
if (!String.IsNullOrEmpty(_options.Value.UserName) && !String.IsNullOrEmpty(_options.Value.Password))
    settings.BasicAuthentication(_options.Value.UserName, _options.Value.Password);
```